test/xspec-catch.xspec

```diff
@@ -19,6 +19,8 @@
 						test="$x:result?err?description treat as xs:string" />
 					<x:expect label="err:value" select="'my', 'error', 'object'"
 						test="$x:result?err?value treat as xs:string+" />
+
+					<!-- The error is raised by fn:error() at a line in the compiled stylsheet/query -->
```

Comment:
Thanks for the clarification here. The fact that the test is directly calling `error` instead of calling a user-written function that calls `error` might have been why I was mistaken earlier about the line and column numbers being reported.

Reply:
I was afraid of this misleading way of writing tests. Surely, it was misleading. Thanks for confirming it.